### PR TITLE
dna updates

### PR DESCRIPTION
**Pull request by fippo**
not done yet, need to resolve the issue with the piggybacking-supposition section.

### CODE REVIEW

File: draft-ietf-xmpp-dna.txt [hunk 1]
```diff
@@ -42,23 +41,22 @@ Status of This Memo
    time.  It is inappropriate to use Internet-Drafts as reference
    material or to cite them other than as "work in progress."
 
-   This Internet-Draft will expire on July 27, 2015.
+   This Internet-Draft will expire on July 30, 2015.
```
Comment (fippo) [inline draft-ietf-xmpp-dna.txt]: am I using a wrong version of xml2rfc?

File: draft-ietf-xmpp-dna.xml [hunk 2]
```diff
@@ -150,8 +150,8 @@
 +-------------TLS NEGOTIATION-------------------+
 |                                               |
 |   B: Server Certificate                       |
-|  [B: Certificate Request]                     |
-|  [A: Client Certificate]                      |
+|   B: Certificate Request                      |
+|   A: Client Certificate                       |
 |                                               |
```
Comment (fippo) [inline draft-ietf-xmpp-dna.xml]: removed the [] since it's not optional

File: draft-ietf-xmpp-dna.xml [hunk 3]
```diff
@@ -274,7 +297,7 @@
         <t>Several simplifying assumptions underlie the happy scenario just outlined:</t>
         <t>
           <list style='symbols'>
```
Comment (fippo) [inline draft-ietf-xmpp-dna.xml]: do we also need to spell out that it's quite possible that Server 1 can't validate the cert of server 2? I.e. I connect to goodadvice.pages.de and get a cert which has expired half a decade ago. What am i supposed to do...

File: draft-ietf-xmpp-dna.txt [hunk 4]
```diff
@@ -575,56 +605,38 @@ Internet-Draft                  XMPP DNA                    January 2015
...
 
-   6.  Server 2 checks the PKIX certificate that Server 1 provided; if
-       it is consistent with the XMPP profile [RFC6120] of the matching
-       rules from [RFC6125], Server 2 accepts that there is a strong
-       domain name association between its stream to Server 1 and the
-       DNS domain name rooms.a.example.
+   1.  Server 2 resolves the DNS domain name rooms.a.example.
+   2.  Server 2 determines this resolves to an IP address and port that
+       it is already connected to.
+   3.  Server 2 determines that the PKIX certif&#1110;cate for that
```
Comment (stpeter) [inline draft-ietf-xmpp-dna.txt]: What is "certif&#1110;cate"? ;-)

File: draft-ietf-xmpp-dna.txt [hunk 5]
```diff
@@ -575,56 +605,38 @@ Internet-Draft                  XMPP DNA                    January 2015
...
-       it is rooms.a.example.
 
-   6.  Server 2 checks the PKIX certificate that Server 1 provided; if
-       it is consistent with the XMPP profile [RFC6120] of the matching
-       rules from [RFC6125], Server 2 accepts that there is a strong
-       domain name association between its stream to Server 1 and the
-       DNS domain name rooms.a.example.
+   1.  Server 2 resolves the DNS domain name rooms.a.example.
+   2.  Server 2 determines this resolves to an IP address and port that
+       it is already connected to.
```
Comment (stpeter) [inline draft-ietf-xmpp-dna.txt]: Do we think that this really necessary?

File: draft-ietf-xmpp-dna.xml (hunk 3 above)
Comment (stpeter) [inline draft-ietf-xmpp-dna.xml]: Yes I think it would be good to discuss that scenario.

File: draft-ietf-xmpp-dna.xml (hunk 2 above)
Comment (stpeter) [inline draft-ietf-xmpp-dna.xml]: +1

File: draft-ietf-xmpp-dna.txt (hunk 5 above)
Comment (fippo) [inline draft-ietf-xmpp-dna.txt]: yes, see http://xmpp.org/extensions/xep-0220.html#targetmultiplex

Or did you refer to the whole process?

File: draft-ietf-xmpp-dna.txt (hunk 4 above)
Comment (fippo) [inline draft-ietf-xmpp-dna.txt]: it's called unicode an evil!

File: draft-ietf-xmpp-dna.xml (hunk 3 above)
Comment (fippo) [inline draft-ietf-xmpp-dna.xml]: in place without adding another section? That would change the section numbering and I am not eager to manually check all cross-references.

Comment (stpeter) [inline draft-ietf-xmpp-dna.xml]: Well the cross-references should be auto-generated in the text file from the XML, so let's not be afraid of a new section. :-)

**Comment by stpeter** [comment]
+1 to merging, what say you @linuxwolf?

**Comment by linuxwolf** [comment]
-----BEGIN PGP SIGNED MESSAGE-----
Hash: SHA512

Just got done scanning through it ... looks good to me, too.  Let's
make it master!

I want to do one more thorough read before submitting the I-D, though.


- -- 
- - m&m

Matthew A. Miller
< http://goo.gl/LK55L >

On 2/2/15 8:05 PM, Peter Saint-Andre wrote:
> +1 to merging, what say you @linuxwolf
> <https://github.com/linuxwolf>?
> 
> — Reply to this email directly or view it on GitHub 
> <https://github.com/linuxwolf/xmpp-fed/pull/2#issuecomment-72584501>.
>
-----BEGIN PGP SIGNATURE-----
Version: GnuPG/MacGPG2 v2.0.22 (Darwin)
Comment: GPGTools - https://gpgtools.org

iQEcBAEBCgAGBQJU0Ow5AAoJEDWi+S0W7cO1Dj0H/jY1CfYIrrkjUPGnv2NhIY6A
aB2lSRD0+EC23Oa3hxVI9RSy+fkggtt8QjgMLNhZ4K5AF2XfFeyv+q04tu/fF+pp
AnxYjLwwNMu6IeYC557lAK+TY7dg7qF1a0YieSC2UZZoELK/Yr4d53Gu54X/yZHf
TAc3H2E++OhxT9LzHC/64kbzpwK1ls8UQcKsIt/QMck3lvIO26MJIvxlPMyAVlZR
cxcP6/siC+3sPmZpHaYB0aBnCZletjBLOjtPVnnBV/GeKYAV55gbWoGCKyJKC8FO
5/Zw9yqQVa4lEetTpyvKcgHmuaPCr+6aAARrhrhU+r+hCylkCNEBcBQQeYSJ4rM=
=LNzE
-----END PGP SIGNATURE-----

**Pull request merged**